modules/api/src/main/java/org/apache/fluo/api/config/SimpleConfiguration.java

```diff
@@ -250,6 +277,22 @@ public SimpleConfiguration subset(String prefix) {
     return new SimpleConfiguration(internalConfig.subset(prefix));
   }
 
+  /**
+   * @param SimpleConfigs SimpleConfiguration's to be merged
+   * @return SimpleConfiguration
+   * 
+   * @since 1.2.0
+   */
+  public static SimpleConfiguration merge(SimpleConfiguration... simpleConfigs) {
+    SimpleConfiguration mrg = new SimpleConfiguration();
+    for (SimpleConfiguration sc : simpleConfigs) {
+      for (Map.Entry<String, String> entry : sc.toMap().entrySet()) {
+        mrg.addProperty(entry.getKey(), entry.getValue());
```

Comment:
@keith-turner I was thinking it would just hang off of SimpleConfiguration, and produce a new SimpleConfiguration, like a linked list.

@cjmctague 'conjoin' probably isn't the best term to use. Also, grammatically, I believe the preposition to use with 'conjoin' is 'with' (mutual relationship) instead of 'to' (directional relationship), as in 'conjoinWith' instead of 'conjoinTo'.